### PR TITLE
JPERF-620: Fix last attempt sleep

### PR DESCRIPTION
**Pull request by dagguh**

### CODE REVIEW

File: src/test/resources/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mzyromski-atlassian) [inline src/test/resources/log4j2.xml:1]: Drive-by change or related to the PR?

**Review by mzyromski-atlassian — COMMENTED** [review]
Could you set JPERF-620 status to indicate it's in review?

**Review by mzyromski-atlassian — DISMISSED** [review]

File: src/test/resources/log4j2.xml (hunk 1 above)
Comment (dagguh) [reply src/test/resources/log4j2.xml:1]: It's related in order to see the changed log messages.

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,11 @@ The API consists of all public Kotlin types from `com.atlassian.performance.tool
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jvm-tasks/compare/release-1.0.1...master
 
+### Fixed
+- Stop sleeping after last failed attempt in `IdempotentAction`. Fix [JPERF-620].
+
+[JPERF-461]: https://ecosystem.atlassian.net/browse/JPERF-461
```
Comment (dagguh) [inline CHANGELOG.md]: Oops, wrong link.
And CHANGELOG moved since then. I'll push an update.

- [x] move the changelog under "unreleased"
- [x] fix the Jira link

**Review by mzyromski-atlassian — APPROVED** [review]
